Reduce audio delay playing Youtube videos.
Patch by lucasvr@gobolinux.org
--- apulse-0.1.8.orig/src/apulse-stream.c	2017-03-08 13:38:10.000000000 -0300
+++ apulse-0.1.8/src/apulse-stream.c	2017-03-08 20:36:17.074256258 -0300
@@ -77,7 +77,7 @@ data_available_for_stream(pa_mainloop_ap
     pa_stream          *s = userdata;
     snd_pcm_sframes_t   frame_count;
     size_t              frame_size = pa_frame_size(&s->ss);
-    char                buf[16 * 1024];
+    char                buf[8 * 1024];
     const size_t        buf_size = pa_find_multiple_of(sizeof(buf), frame_size);
     int                 paused = g_atomic_int_get(&s->paused);
 
@@ -593,7 +593,7 @@ pa_stream_new_with_proplist(pa_context *
     s->timing_info.configured_source_usec = 0;
     s->timing_info.since_underrun = 0;
 
-    s->rb = ringbuffer_new(72 * 1024);    // TODO: figure out size
+    s->rb = ringbuffer_new(8 * 1024);    // TODO: figure out size
     s->peek_buffer = malloc(s->rb->end - s->rb->start);
 
     return s;
@@ -754,7 +754,7 @@ pa_stream_writable_size(pa_stream *s)
     //
     // Workaround issue by reserving certain amount for that case.
 
-    const size_t limit = 16 * 1024; // TODO: adaptive values?
+    const size_t limit = 8 * 1024; // TODO: adaptive values?
 
     if (writable_size < limit)
         writable_size = 0;
